### ISSUE TITLE
DT package is used but not mentioned in DESCRIPTION

### ISSUE THREAD

**Issue by HeidiSeibold** [daattali/shinyforms#22]
I was just trying shinyforms and when running your example I got

```
> ui <- fluidPage(
+ formUI(formInfo)
+ )
Error in loadNamespace(name) : there is no package called ‘DT’
```

No hurry though. I think people can figure it out for now :wink:

If they don't they will probably find this and see that the solution is `install.packages('DT')`

**Comment by daattali** [comment]
Thanks!  (It makes me really uneasy to know that many people are already trying to use this package -- it's very very not ready!)

**Comment by daattali** [comment]
fixed via 589625ae2cb59ee76c64864b3c1688df79309983
